addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -70,6 +71,15 @@ export class SearchBar extends Component {
 
         const tasks = [];
         for (const id of expanded) {
+            // expand a properties should lazily fetch the definition, and generate
+            // searchItems, so the normal field logic is used for properties
+            const searchItem = this.searchItems.find((i) => i.id === id);
+            const field = this.fields[searchItem.fieldName];
+            if (field.type === "properties" && searchItem.type !== "field_property") {
```

Comment:
Speaking of duplicating this condition ^^

Can "searchItem.type" even be a field_property at this point?
Since we are currently adding them if I understand correctly?

If we only need the type, maybe we could filter them out as in my previous comments, that way we only loop on expanded properties items and make it very simple.